server/alerts_email_via_smtp.py

```diff
@@ -0,0 +1,270 @@
+padding:3.75pt 3.75pt 3.75pt 3.75pt;background:#EEEEEE'><p class=MsoNormal><code><span style='font-size:10.0pt'>
+{{ template_vars.time }}</span></code><span style='font-size:10.5pt'><o:p></o:p></span></p></td></tr><tr><td width=180 
+style='width:135.0pt;border:solid #414747 1.0pt;border-top:none;background:#EEEEEE;padding:3.75pt 3.75pt 3.75pt 3.75pt;_border: none'>
+<p class=MsoNormal><b><span style='font-size:10.5pt'>Campaign Expiration<o:p></o:p></span></b></p></td>
+<td style='border-top:none;border-left:none;border-bottom:solid #414747 
+1.0pt;border-right:solid #FFC20E 1.0pt;padding:3.75pt 3.75pt 3.75pt 3.75pt;background:#EEEEEE'><p class=MsoNormal><code>
+<span style='font-size:10.0pt'>{{ template_vars.expiration }}</span></code><span style='font-size:10.5pt'><o:p></o:p>
+</span></p></td></tr></table></td></tr><tr><td style='padding:.75pt .75pt .75pt .75pt'>   <tr>
+<td width="100%" style='width:100.0%;border:none;padding:7.5pt .15in 7.5pt .15in;min-width: 315px;max-width: 660px'>
+<div align=center><table class=MsoNormalTable border=0 cellspacing=3 cellpadding=0 width="100%" style='width:100.0%'><tr>
+<td style='padding:.75pt .75pt .75pt .75pt;color:white'><p class=MsoNormal>Powered by: 
+<a href="https://github.com/securestate/king-phisher">RSM US, LLP</a> 
+<o:p></o:p></p></td></tr></table></div></td></tr></table><div class="Footer">
+<img src="https://github.com/securestate/king-phisher/raw/master/data/king-phisher-logo.png">
+</div>
+<p class=MsoNormal><o:p>&nbsp;</o:p></p></div></body></html>
+"""
+
+class Plugin(plugins.ServerPlugin):
+	authors = ['Austin DeFrancesco', 'Spencer McIntyre', 'Mike Stringer', 'Erik Daguerre']
+	classifiers = ['Plugin :: Server :: Notifications :: Alerts']
+	title = 'Campaign Alerts: via Python 3 SMTPLib'
+	description = """
+	Send campaign alerts via the SMTP Python 3 lib. This requires that users specify
+	their email through the King Phisher client to subscribe to notifications.
+	"""
+	homepage = 'https://github.com/securestate/king-phisher-plugins'
+
+	# Email accounts with 2FA, such as Gmail, will not work unless "less secure apps" are allowed
+	# Reference: https://support.google.com/accounts/answer/60610255
+	# Gmail and other providers require SSL on port 465, TLS will start with the activation of SSL
+	options = [
+		plugin_opts.OptionString(
+			name='smtp_host',
+			description='Location of SMTP server',
+			default='localhost'
+		),
+		plugin_opts.OptionInteger(
+			name='smtp_port',
+			description='Port used for SMTP server',
+			default=25
+		),
+		plugin_opts.OptionString(
+			name='smtp_email',
+			description='SMTP email address to send notifications from',
+			default=''
+		),
+		plugin_opts.OptionString(
+			name='smtp_password',
+			description='Password associated with smtp email',
+			default=''
+		),
+		plugin_opts.OptionBoolean(
+			name='ssl',
+			description='Set connection to smtp server to use ssl connection',
+			default=True
```

Comment:
This should be false to be consistent with the default port of 25. Also since this has a default, please don't use `self.config.get('ssl',...` in the plugin.